tests/models/test_modeling_common.py

```diff
@@ -565,3 +574,79 @@ def test_deprecated_kwargs(self):
                 f" {self.model_class}.__init__ if there are deprecated arguments or remove the deprecated argument"
                 " from `_deprecated_kwargs = [<deprecated_argument>]`"
             )
+
+
+@is_staging_test
+class ModelPushToHubTester(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._token = TOKEN
+        HfFolder.save_token(TOKEN)
```

Comment:
This will save the token on disk when running the test. I don't think it is necessary if you are already passing the token as input parameter in `create_repo`, `delete_repo`, `push_to_hub`,... The issue with saving the token on disk is that it will impact all the other tests as well.

Also in general, it's better to do `login(token=token)` rather than `HfFolder.save_token(token)`. It is more explicit what the line does (+it's closer to what a real user would do).